zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -56,5 +56,7 @@ pub(crate) const N_BYTES_PROGRAM_COUNTER: usize = N_BYTES_U64;
 // Number of bytes that will be used for a tx's gas field.
 pub(crate) const N_BYTES_GAS: usize = N_BYTES_U64;
 
+pub(crate) const N_BYTES_STACK: usize = 2;
+
```

Comment:
i think this is only useful in ErrorStackGadget. So maybe put it in that file is good enough. Don't need to be put it  high level.

Reply:
move to gadget level in [3c9c55d](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/873/commits/3c9c55da2df49e8f6a236241b8d42fcfe6f4f465) thanks!